wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/GenericHIDSim.java

```diff
@@ -118,7 +118,7 @@ public void setName(String name) {
   }
 
   /**
-   * Set the type of an axis.
+   * Set the type of the provided axis based on axis channel.
```

Comment:
```suggestion
   * Set the type of the provided axis channel.
```
I think that would read better